Readme.md

```diff
@@ -75,9 +75,14 @@ Usage:
 const jsTargets = require('path/to/Lombiq.Gulp.Extensions/Tasks/js-targets');
 
 const source = './Assets/Scripts/'
-const destination = './directory-to-copy-into'
+const destination = './wwwroot/js'
 
 gulp.task('build:js', () => jsTargets.compile(source, destination));
+
+// Or you can pass a function to modify the pipeline before saving the files and e.g. run Babel:
+gulp.task(
+    'build:js',
+    () => jsTargets.compile(source, destination, (pipeline) => pipeline.pipe(babel({ presets: ['@babel/preset-env'] }))));
```

Comment:
I assume, the () around pipeline are a convention, so should stay? Because, they should not be necessary.

Reply:
Yes. Like ending enumerations of items with a `,` this makes diffing a bit nicer: https://github.com/airbnb/javascript#arrows--one-arg-parens